src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheck.java

```diff
@@ -269,44 +271,44 @@
     /**
      * Define the annotation element styles.
      */
-    private ElementStyle elementStyle = ElementStyle.COMPACT_NO_ARRAY;
+    private ElementStyleOption elementStyleOption = ElementStyleOption.COMPACT_NO_ARRAY;
 
     // defaulting to NEVER because of the strange compiler behavior
     /**
      * Define the policy for trailing comma in arrays.
      */
-    private TrailingArrayComma trailingArrayComma = TrailingArrayComma.NEVER;
+    private TrailingArrayCommaOption trailingArrayCommaOption = TrailingArrayCommaOption.NEVER;
 
     /**
      * Define the policy for ending parenthesis.
      */
-    private ClosingParens closingParens = ClosingParens.NEVER;
+    private ClosingParensOption closingParensOption = ClosingParensOption.NEVER;
 
     /**
      * Setter to define the annotation element styles.
      *
      * @param style string representation
      */
-    public void setElementStyle(final String style) {
-        elementStyle = getOption(ElementStyle.class, style);
+    public void setElementStyleOption(final String style) {
```

Comment:
Property names are not expected to be changed. This will break configurations.

Reply:
Done.